### PR TITLE
Final student show (Also added background image)

### PR DESCRIPTION
**Pull request by coreycartercodes**
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please describe):

## Problem/feature

_What problem are you trying to solve?_
Add to show page

## Solution
integrated 'points' API call

## Other changes (e.g. bug fixes, UI tweaks, small refactors)
- Added background image
## Checklist
- [X] Code compiles correctly
- [X] Tests for the changes have been added (for bug fixes/features)
- [X] All tests passing
- [ ] No rubocop offenses
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by coreycartercodes** [comment]
Looks like Travis hates bootstrap. Will have to look into that one!

**Comment by priyapower** [comment]
@coreycartercodes This looks great for the controller and the views, however, this didn't have the points spec for a test. I pulled down and added one. It is tested and pushed up. I believe it is ready for a merge with the exception of mocking our requests

File: spec/features/students/courses/points_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Student Points' do
+  scenario 'a student can see their points on their course show page' do
```
Comment (priyapower) [inline spec/features/students/courses/points_spec.rb:4]: Added this spec test to confirm points are on page

**Review by priyapower — APPROVED** [review]
Looks great

**Review by sarynm12 — APPROVED** [review]

**Review by Arique1104 — APPROVED** [review]

**Pull request merged**